### DISCUSSION TITLE
ShadowJar error

### DISCUSSION THREAD

**Discussion by denmeh** [denmeh/holoeasy#71] [Q&A]
@GeorgeV220 In the last commit if I execute a shadowJar on the holoeasy-example-packetevents, if I put implementation (project(":holoeasy-core")) the jar goes to 91MB. Any help?

Ps: until it does not get fixed i can not go on on other task ask on the #67 or Folia support discussion because I can not test the library.

**Comment by GeorgeV220** [comment]
give me some time to check it out because I think I made some mistakes!!

**Comment by GeorgeV220** [comment]
@unldenis Check #72
